src/ejabberd_c2s.erl

```diff
@@ -272,8 +267,13 @@ wait_for_stream(closed, StateData) ->
 
 handle_stream_start({xmlstreamstart, _Name, Attrs}, #state{} = S0) ->
     Server = jid:nameprep(xml:get_attr_s(<<"to">>, Attrs)),
+    ?ERROR_MSG("Server is ~p; MYNAME is ~p~n", [Server, ?MYNAME]),
```

Comment:
WTF? Error level?! :D

Reply:
FUCK, that's me trying to see the value. and forgetting to remove it when committing xD